test/testdata/config_apis_test/golden.txt

```diff
@@ -1,104 +0,0 @@
-<!-- Generated with Stardoc: http://skydoc.bazel.build -->
```

Comment:
did you mean to delete this?

Reply:
Oof. My script didn't catch that this got blatantly removed because for this test is missing a target! I've added the target.